owasm/src/ext/bitcoin/block_count.rs

```diff
@@ -0,0 +1,47 @@
+//! [blockchain.info] (https://blockchain.info/q/getblockcount) Oracle Extension
+
+use crate::core::{Oracle, ShellCmd};
+use crate::ext::utils::curl::Curl;
+
+pub struct Info {}
+
+impl Info {
+    pub fn new() -> Info {
+        Info {}
+    }
+}
+
+impl Oracle for Info {
+    type T = u64;
+
+    fn as_cmd(&self) -> ShellCmd {
+        Curl::new(&["https://blockchain.info/q/getblockcount"]).as_cmd()
+    }
+
+    fn from_cmd_output(&self, output: String) -> Option<u64> {
+        json::parse(&output).ok()?.as_u64()
```

Comment:
Output is `616081` we need to use json parser?

Reply:
done